changelog/_go-ver-1160.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:change
-core: Bump Go version to 1.21.8.
+core: Bump Go version to 1.21.9.
```

Comment:
Should be in a new file `changelog/_go-ver-1162.txt`

Reply:
Oh shoot, whoops! I messed this up on the other release branches. Will fix on all, thanks!